### PR TITLE
Update dependencies for compatibility between firestore and mapquest

### PR DESCRIPTION
**Pull request by TheImpressiveDonut**
problem with protobuf package between firestore and mapquest

### CODE REVIEW

File: app/src/main/java/com/github/onedirection/database/utils/FirebaseUtils.java [hunk 1]
```diff
@@ -13,7 +14,13 @@ public static FirebaseFirestore getFirestore() {
         if (firestore == null) {
             firestore = FirebaseFirestore.getInstance();
             if (useEmulator) {
-                firestore.useEmulator("10.0.2.2", 8080);
```
Comment (justagoodguy-omega) [inline app/src/main/java/com/github/onedirection/database/utils/FirebaseUtils.java:16]: Maybe donc hardcode values like this, you can use string.xml instead for example

**Review by justagoodguy-omega — CHANGES_REQUESTED** [review]

**Review by justagoodguy-omega — APPROVED** [review]

File: app/build.gradle [hunk 2]
```diff
@@ -47,14 +58,21 @@ dependencies {
     implementation 'androidx.cardview:cardview:1.0.0'
     implementation 'com.google.android.material:material:1.4.0-alpha02'
     implementation 'androidx.annotation:annotation:1.2.0'
-    implementation platform('com.google.firebase:firebase-bom:26.6.0')
-    implementation 'com.google.firebase:firebase-auth'
-    implementation 'com.google.firebase:firebase-firestore'
+    // firebases library version are changed to be compatible with mapquest
+    //noinspection GradleDependency
+    implementation 'com.google.firebase:firebase-auth:20.0.3'
+    //noinspection GradleDependency
```
Comment (Ef55) [inline app/build.gradle:64]: Leftovers. Or does gradle read comments ?

**Review by Ef55 — CHANGES_REQUESTED** [review]

File: app/build.gradle (hunk 2 above)
Comment (TheImpressiveDonut) [reply app/build.gradle:64]: So the noinspection is for the lint warning since there are new version available

Comment (TheImpressiveDonut) [reply app/build.gradle:64]: For gradle it is just a comment, it's an instruction for the linter

Comment (Ef55) [reply app/build.gradle:64]: So it does indeed read the comments (but not gradle).
Thanks

**Review by Ef55 — APPROVED** [review]

**Pull request merged**